opengrok-web/pom.xml

```diff
@@ -256,21 +256,48 @@ Portions Copyright (c) 2018, 2020, Chris Fraire <cfraire@me.com>.
                     </argLine>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>com.github.blutorange</groupId>
+                <artifactId>closure-compiler-maven-plugin</artifactId>
+                <version>2.21.0</version>
+                <configuration>
+                    <baseSourceDir>${project.basedir}/src/main/webapp</baseSourceDir>
+                    <baseTargetDir>${project.build.directory}/source</baseTargetDir>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>default-minify</id>
+                        <configuration>
+                            <encoding>UTF-8</encoding>
+                            <sourceDir>js</sourceDir>
+                            <targetDir>js</targetDir>
+                            <includes>**/*.js</includes>
+                            <excludes>**/*.min.js</excludes>
+                            <skipMerge>true</skipMerge>
+                            <closureLanguageOut>ECMASCRIPT5</closureLanguageOut>
+                        </configuration>
+                        <goals>
+                            <goal>minify</goal>
+                        </goals>
+                        <phase>generate-resources</phase>
+                    </execution>
+                </executions>
+            </plugin>
             <plugin>
                 <groupId>net.alchim31.maven</groupId>
                 <artifactId>yuicompressor-maven-plugin</artifactId>
                 <version>1.5.1</version>
                 <executions>
                     <execution>
                         <goals>
-                            <goal>jslint</goal>
```

Comment:
does the closure compressor provide JS lint functionality or are we going to rely on the Github's checks ?

Reply:
I've tried to set https://blutorange.github.io/closure-compiler-maven-plugin/minify-mojo.html#closureWarningLevel to `VERBOSE` but it failed the build and I did not immediately see an option which would set treating errors as warnings. It would make sense to try to actually fix the errors. I'll try to look into it.